examples/kitchensink-ts/package.json

```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "@strapi/plugin-i18n": "4.1.12",
```

Comment:
You will also need to upgrade the plugins versions to be 4.3.2

Reply:
Done in bd6acfdd9052a737787fde1a3d145404acfecb80